components/observations/ObservationsListView.tsx

```diff
@@ -95,23 +127,26 @@ export const ObservationsListView: React.FunctionComponent<ObservationsListViewP
             </TouchableOpacity>
           </HStack>
         }
-        ListFooterComponent={
-          nacObservationsResult.hasNextPage || nwacObservationsResult.hasNextPage ? (
-            <HStack justifyContent="center" mt={4}>
-              <Button
-                width={'50%'}
-                buttonStyle={'primary'}
-                disabled={nacObservationsResult.isFetchingNextPage || nwacObservationsResult.isFetchingNextPage}
-                busy={nacObservationsResult.isFetchingNextPage || nwacObservationsResult.isFetchingNextPage}
-                onPress={() => {
-                  void nwacObservationsResult.fetchNextPage();
-                  void nacObservationsResult.fetchNextPage();
-                }}>
-                <BodyBlack>Load more...</BodyBlack>
-              </Button>
-            </HStack>
-          ) : null
-        }
+        ListFooterComponent={() => {
+          if (!nacObservationsResult.hasNextPage && !nwacObservationsResult.hasNextPage) {
+            return (
+              <Center height={80}>
+                <Body>
+                  {/* this is super-sketchy, need to implement proper string localization/formatting */}
```

Comment:
Can we get an issue to document how you would like this to be done in a proper way?

Reply:
Good call, created #329